cmd/README.md

```diff
@@ -0,0 +1,521 @@
+
+## Example Usage
+
+Details on HarbourBridge example usage can be found here: 
+- [PostgreSQL example usage](sources/postgres/README.md#example-postgresql-usage)
+- [MySQL example usage](sources/mysql/README.md#example-mysql-usage)
+- [DynamoDB example usage](sources/dynamodb/README.md#example-dynamodb-usage)
+
+
+## Schema Conversion
+
+Details on HarbourBridge schema conversion can be found here:
+- [PostgreSQL schema conversion](sources/postgres/README.md#schema-conversion)
+- [MySQL schema conversion](sources/mysql/README.md#schema-conversion)
+- [DynamoDB schema conversion](sources/dynamodb/README.md#schema-conversion)
+
+## Data Conversion
+
+HarbourBridge converts PostgreSQL/MySQL/DynamoDB data to Spanner data based on 
+the Spanner schema it constructs. Conversion for most data types is fairly 
+straightforward, but several types deserve discussion. Note that HarbourBridge 
+is not intended for databases larger than about 100GB. Details on HarbourBridge 
+data conversion can be found here:
+- [PostgreSQL data conversion](sources/postgres/README.md#data-conversion)
+- [MySQL data conversion](sources/mysql/README.md#data-conversion)
+- [DynamoDB data conversion](sources/dynamodb/README.md#data-conversion)
+
+## Troubleshooting Guide
+
+HarbourBridge is written using the Go module system, and so it must be
+installed in [module-aware mode](https://golang.org/cmd/go/#hdr-Module_support).
+This can be achived by setting the environment variable `GO111MODULE` to `on`
+before calling "go get" e.g.
+
+```sh
+GO111MODULE=on go get github.com/cloudspannerecosystem/harbourbridge
+```
+
+As an alternative to "go get", you can make a copy of the HarbourBridge
+codebase from the github repository and use "go run":
+
+```sh
+git clone https://github.com/cloudspannerecosystem/harbourbridge
+cd harbourbridge
+pg_dump mydb > mydb.pg_dump
+go run github.com/cloudspannerecosystem/harbourbridge schema -source=postgresql < mydb.pg_dump
+```
+
+This workflow also allows you to modify or customize the HarbourBridge codebase.
+
+The following steps can help diagnose common issues encountered while running
+HarbourBridge.
+
+### 1. Verify driver configuration
+
+First, check that the driver is correctly configured to connect to your
+database. Driver configuration varies depending on the driver used.
+
+#### 1.1 pg_dump
+
+If you are using pg_dump (-driver=pg_dump), check that pg_dump is
+correctly configured to connect to your PostgreSQL
+database. Note that pg_dump uses the same options as psql to connect to your
+database. See the [psql](https://www.postgresql.org/docs/9.3/app-psql.html) and
+[pg_dump](https://www.postgresql.org/docs/9.3/app-pgdump.html) documentation.
+
+Access to a PostgreSQL database is typically configured using the
+_PGHOST_, _PGPORT_, _PGUSER_, _PGDATABASE_ environment variables,
+which are standard across PostgreSQL utilities.
+
+It is also possible to configure access via pg_dump's command-line options
```

Comment:
These flags will move into the source profile

Reply:
I wasn't sure what this means, but wonder if this is also something that should be cleaned up separately.